src/LoRa_APRS_Tracker.cpp

```diff
@@ -131,7 +138,28 @@ void setup() {
   batteryVoltage = String(getBattVoltage());
   // EEPROM.begin(512);
 }
-
+void telemetry() {
+  pinMode(Vext, OUTPUT);
+  digitalWrite(Vext, LOW);
+  Serial.begin(115200);
```

Comment:
Serial was already started in Setup

Reply:
Was having some issues before with the hardware, might have left that in by mistake when I was testing.